publication-testing/src/main/java/no/unit/nva/publication/TestDataSource.java

```diff
@@ -0,0 +1,23 @@
+package no.unit.nva.publication;
+
+import no.unit.nva.model.Publication;
+
+import static no.unit.nva.model.testing.PublicationGenerator.randomPublication;
+import static nva.commons.core.attempt.Try.attempt;
+
+public class TestDataSource {
+
+    public Publication publicationWithIdentifier() {
+        return randomPublication();
+    }
+
+    public Publication publicationWithoutIdentifier() {
+        var publication = randomPublication();
+        publication.setIdentifier(null);
+        return publication;
+    }
```

Comment:
were these methods moved from somewehere?

Reply:
Yes, there are two methods in PublicationGenerator that are not used anywhere (or tested) in that project.